module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,59 @@
+#include "compression.hpp"
+#include <algorithm>
+#include <iostream>
+#include <iomanip>
+
+void printMap (std::array<std::array<uint8_t, width>, height> bitmap)
+{
+    for(const auto& line : bitmap)
+    {
+        for (const auto c : line)
+        {
+            std::cout<<std::setw(3)<<(int)c<<' ';
+
```

Comment:
redundant enter

Reply:
I did not get it :). Can you please explain?